src/length-value.js

```diff
@@ -44,19 +44,31 @@
   LengthValue.prototype = Object.create(shared.StyleValue.prototype);
 
   // Length Calculation Methods
-  LengthValue.prototype.add = function(value) {
-    throw new TypeError('Not implemented yet');
+  LengthValue.prototype.add = function(addedLength) {
+    if(this instanceof SimpleLength && addedLength instanceof SimpleLength){
+      if(this.type === addedLength.type){
+        return this.addSimpleLengths(addedLength);
+      }
```

Comment:
You should collapse these two if statements into one, or add an else case to the inner one. At the moment it's possible to return undefined.